src/com/itacademy/eshop/interfaces/Expiring.java

```diff
@@ -0,0 +1,7 @@
+package com.itacademy.eshop.interfaces;
+
+public interface Expiring {
```

Comment:
Good idea. This is just for demonstration purposes so it is okay, but keep in mind that you usually create interface when there is a need for it in at least two places.